DisguiseUnityRenderStream/Runtime/PipelineAbstraction/DepthCopy.cs

```diff
@@ -8,7 +8,7 @@ namespace Disguise.RenderStream
     /// <summary>
     /// Copies a camera's depth into a texture.
     /// </summary>
-    class DepthCopy
+    public class DepthCopy
```

Comment:
With this new assembly separation, you don't have to make any of the classes `public`. You can just use the `InternalsVisibleTo` attribute in the `Diguise.RenderStream` assembly.